src/GeneExpressionTSnePlot.js

```diff
@@ -96,14 +96,15 @@ const GeneExpressionScatterPlot = (props) => {
   const colourSchema = [`#d4e4fb`,`#95adde`,`#6077bf`,`#1151D1`,`#35419b`,`#0e0573`] // light blue to dark blue
   const colourSchemaLength = colourSchema.length
 
-  const plotDisable = !plotData.max
+  const plotIsDisabled = !plotData.max
 
-  const dataScale = plotDisable ?
+  const dataScale = plotIsDisabled ?
     0 :
-    plotData.max.toFixed(0).toString().length // The digit before demical
+    plotData.max.toFixed(0).toString().length // Number of digits before decimal point
   const highchartsConfig = {
     chart: {
-      height: plotDisable ? height * 0.95 : height
+      // Magic number to adjust the height discrepancy between the two charts
```

Comment:
Out of curiosity, where does this 12 come from? Is this because the search box on top isn’t a React Select component?

Reply:
I think it's because his new adapt to height plugin adjusts the height of the chart on the left  based on the height of the legend (which is about 12px), but the chart on the right doesn't have a legend so it stays at the initial height. I think....

Reply:
No, that's not true because the magic 12 works even when legends are more than 1 line. I don't know anymore... Highcharts magic. 🙃